fem/datacollection.cpp

```diff
@@ -637,7 +645,8 @@ void VisItDataCollection::LoadFields()
                it->first,
                new ParGridFunction(dynamic_cast<ParMesh*>(mesh), file), own_data);
          }
-         else if ((it->second).association == "elements")
+         else if ((it->second).association == "elements" || // old style
+                  (it->second).association == "quadrature") // new style
```

Comment:
This seems safe for backward compatibly but would be good for others to think and confirm